editor/editor_themes.cpp

```diff
@@ -143,7 +143,8 @@ Ref<Theme> create_editor_theme() {
 	Color light_color_1 = base_color.linear_interpolate(Color(1, 1, 1, 1), contrast);
 	Color light_color_2 = base_color.linear_interpolate(Color(1, 1, 1, 1), contrast * 1.5);
 
-	const int border_width = (border_size % 3) * EDSCALE;
+	const int optional_border_width = (border_size % 3) * EDSCALE;
+	const int border_width = optional_border_width == 0 ? EDSCALE : optional_border_width;
```

Comment:
The names don't make much sense to me. `border_width` in only used for two widgets, and `optional_border_width` for the others; in what way is it "optional"?

Also the logic looks a bit weird, `border_size == 3` or `6` or `9` would mean no `optional_border_width`? And actually border_size in [0, 1, 3, 4, 6, 7, 9, 10, etc.] would all result in the same behaviour... I might be missing something but it looks like a convoluted way to have a random distribution with 2/3 of probabilities for one value and 1/3 for the other :)